### DISCUSSION TITLE
How to show on table recently added data without refreshing the page

### DISCUSSION THREAD

**Discussion by NeverioDev** [rappasoft/laravel-livewire-tables#730] [Q&A]
After adding the data, the table doesn't show it automatically, I have to refresh the page to see changes

**Comment by hariadi** [comment]
Emit event: `$this->emit('refreshDatatable');`

See: https://github.com/rappasoft/laravel-livewire-tables/blob/v2.1.1/src/DataTableComponent.php#L40

**Reply by qwasyx0** [reply]
This doesn't work for me https://github.com/rappasoft/laravel-livewire-tables/discussions/766
